environment-dev.yml

```diff
@@ -2,21 +2,9 @@ name: landlab_dev
 channels:
   - conda-forge
 dependencies:
-  # for development
 - black
 - flake8
 - isort
-- pytest
-- pytest-cov
-- pytest-datadir
-# for the package
-- matplotlib
-- netcdf4
-- pyshp
-- pyyaml
-- scipy
-- statsmodels
-- pandas
-- xarray
 - pip:
-    - bmipy<1
+  - -r file:requirements.txt
```

Comment:
@mcflugen does this form mean that these dependencies will be installed with pip instead of with conda-forge?

I also think that we need this file to include the notebook dependencies (I'm pretty sure that it doesn't at present)

Reply:
Yes, this means install everything that's in *requirements.txt* with *pip*. I don't think I've listed *jupyter* as a requirement anywhere, so, yes, we may need to add it.

Reply:
Don't we want to use conda-forge for as many things as possible?